packages/blade/src/components/Badge/StyledBadge.native.tsx

```diff
@@ -1,8 +1,9 @@
 import styled from 'styled-components/native';
 import { getStyledBadgeStyles } from './getStyledBadgeStyles';
 import type { StyledBadgeProps } from './types';
+import BaseBox from '~components/Box/BaseBox';
 
-const StyledBadge = styled.View<StyledBadgeProps>((props) => ({
+const StyledBadge = styled(BaseBox)<StyledBadgeProps>((props) => ({
```

Comment:
Actually we're doing this at several places. Just want to make minimal unrelated changes in this PR for now since it is already too big. Lets refactor these things some other time separately?

Reply:
create a follow up issue for this so we can iron this out?

Reply:
Here - https://github.com/razorpay/blade/issues/1046